xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -39,7 +39,7 @@
   
  Standard input is usually the keyboard, and standard output and standard error are usually the monitor screen. However, you can use the <xref:System.Diagnostics.ProcessStartInfo.RedirectStandardInput%2A>, <xref:System.Diagnostics.ProcessStartInfo.RedirectStandardOutput%2A>, and <xref:System.Diagnostics.ProcessStartInfo.RedirectStandardError%2A> properties to cause the process to get input from or return output to a file or other device. If you use the <xref:System.Diagnostics.Process.StandardInput%2A>, <xref:System.Diagnostics.Process.StandardOutput%2A>, or <xref:System.Diagnostics.Process.StandardError%2A> properties on the <xref:System.Diagnostics.Process> component, you must first set the corresponding value on the <xref:System.Diagnostics.ProcessStartInfo> property. Otherwise, the system throws an exception when you read or write to the stream.  
   
- Set <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> to specify whether to start the process by using the operating system shell.  
+ Set <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> to specify whether to start the process by using the operating system shell. If <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> is set to false, the new process will inherit the standard input, standard output, and standard error streams of the calling process, unless <xref:System.Diagnostics.ProcessStartInfo.RedirectStandardInput%2A>, <xref:System.Diagnostics.ProcessStartInfo.RedirectStandardOutput%2A>, or <xref:System.Diagnostics.ProcessStartInfo.RedirectStandardError%2A> are set to true. 
```

Comment:
inline code the keywords false and true to avoid having them translated like this: \`false\`

will inherit -> inherits (use present tense whenever possible)

also add respectively in the end

it would also be good to identify what these are. For example: 
- Set **the** <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> **property** to `true` to specify whether to start the process by using the operating system shell
- unless **the** RedirectStandardInput, RedirectStandardOutput and RedirectStandardError **properties**

Reply:
@mairaw I have updated the wording, does it look better now?